### ISSUE TITLE
SCM: changes title bar buttons/actions are not accessible in multi repositories

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#101432]
Testing #101302

When there is only one repository in the workspace and also no other extensions contributing to custom views to SCM, I can tab between Input Box, Changes title bar buttons:

![image](https://user-images.githubusercontent.com/876920/86153263-3db04d80-bab6-11ea-8b1e-17dc74934986.png)

However, when there are multiple views, pressing Tab in Input Box will jump immediately to the next Repository view's title as buttons on Changes title bar is now invisible.

![image](https://user-images.githubusercontent.com/876920/86153361-6c2e2880-bab6-11ea-887f-3873d4cfd0ba.png)

**Comment by joaomoreno** [comment]
No, I don't think this is accurate... In your first screenshot that only worked because the actions were visible because the `Changes` row was selected.

You can still reach the actions by <kbd>Shift Tab</kbd> until you hit any action or the actual list and the <kbd>UpArrow</kbd> or <kbd>DownArrow</kbd> so you can select the row you want and then <kbd>Tab</kbd> to reach the actions.

One thing we can improve is to focus the list when pressing <kbd>Escape</kbd> inside the commit box. I'll push that.
